### PR TITLE
phone number validation

### PR DESCRIPTION
**Pull request by Georgio-Khoury**

### CODE REVIEW

**Comment by Hkataya** [comment]
@Georgio-Khoury is this ready for review?

File: package.json [hunk 1]
```diff
@@ -19,16 +19,7 @@
...
-  },
-  "dependencies": {
-    "@types/bcryptjs": "^2.4.6",
-    "@types/jsonwebtoken": "^9.0.6",
-    "@types/passport-jwt": "^4.0.1",
-    "bcryptjs": "^2.4.3",
-    "jsonwebtoken": "^9.0.2",
-    "passport": "^0.7.0",
-    "passport-jwt": "^4.0.1",
-  }
```
Comment (samish-dev) [inline package.json:31]: why are we removing these dependancies in this pr?

**Review by samish-dev — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Georgio-Khoury) [reply package.json:31]: we don't want deps in the root package.json

Comment (Hkataya) [reply package.json]: we moved them to the backend package.json as they are only used in backend

**Pull request merged**